### ISSUE TITLE
Enable `CLIENT_FOUND_ROWS` in `r2dbc-mysql` in favor of `spring-data-r2dbc`

### ISSUE THREAD

**Issue by ks-yim** [jasync-sql/jasync-sql#240]
After https://github.com/spring-projects/spring-data-r2dbc/issues/253, `spring-data-r2dbc` assumes `CLIENT_FOUND_ROWS` capability flag is enabled for its mysql connections.

This causes a problem with `jasync-r2dbc-mysql` as a backing driver for `spring-data-r2dbc` when we try to save an unchanged entity...:
```kotlin
val fooRepository: R2dbcRepository<FooEntity, Int>

// throws an exception since `affectedRows` evaluates to 0 in MySQL without `CLIENT_FOUND_ROWS` flag.
fooRepository.save(fooRepository.findById(1))
```

It would be nice if there's an option to enable `CLIENT_FOUND_ROWS` flag in [MySqlConnection.kt](https://github.com/jasync-sql/jasync-sql/blob/master/mysql-async/src/main/java/com/github/jasync/sql/db/mysql/MySQLConnection.kt) to enable it in [MySqlConnectionFactoryProvider#create](https://github.com/jasync-sql/jasync-sql/blob/master/r2dbc-mysql/src/main/java/MysqlConnectionFactoryProvider.kt#L39-L50).

**Comment by oshai** [comment]
Thanks for raising the issue. I'll need to investigate how to implement that, and will try to do that when possible.

**Comment by ks-yim** [comment]
It seems there's no easy way of overriding capability flags. We can add related properties to `Configuration` class but it feels awkward since they only have meanings in `MySQL` context.

One potential hack is to detect a `spring-data-r2dbc`(or `R2DBC`) related class in the classpath and cache the result upon class loading and conditionally enable `CLIENT_FOUND_FOWS` flag depending on the result.

**Comment by oshai** [comment]
created PR #247

**Comment by oshai** [comment]
Fix is in version 1.2.3.

**Comment by ks-yim** [comment]
Thanks!
